source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+Put another way, what are your desired outcomes?
+
+This takes a small amount of time up front. But, we’ve seen the difference this makes in successful projects. 
+
+Below, we’ve added some key questions to ask yourself, your team, and any other stakeholders.
+
+- Why have we decided to deploy the Mattermost mobile application?
+- Who will, or should, play a key/critical role in this project?
+- Who does this project impact?
+- How much time do we have for this deployment?
+- Should, or could, an initial deployment take a simpler path?
+- Can we stagger our deployment, starting with simple requirements first?
+- Are you going to  run a pilot deployment, if so who with?
+- What  IT or HR policies that need to change?
+
+We recommend setting at least one, achievable and time-based outcome.
+
+At the same time, you should begin thinking about technical and security requirements. A great place to start is with an implementation plan. So, feel free to `use our template available <https://docs.mattermost.com/getting-started/implementation_plan.html>`_ in the Mattermost Documentation. Also, begin by engaging with your technical and security questions like those below.
+
+- Are there any known security or access requirements?
+- Is a VPN connection required?
+- Do we use an Enterprise Mobile Management (EMM) Provider?
+
+Again, this means pausing before you jump into your Mattermost mobile deployment. This takes a little more time, but will make the difference in ensuring your see a return on your investment.
+
+----
+
+###################################
+Mattermost Mobile Deployment Primer
+###################################
+
+A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
+
+We’re here to support you.
+
+This primer serves as an introduction to Mattermost Mobile technologies. It is a compliment to our technical documentation, and focuses on three main areas:
+
+- Technical Requirements
+- Basic Features and Technologies
+- Trouble Spots
+
+Regardless of your expertise, we recommend reading this section before starting your deployment.
+
+***********************
+Technical  Requirements
+***********************
+
+`Supported Mattermost Server Versions <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`_ 
+    Minimum requirements for the Mattermost Server is maintained in our documentation (link above). 
+    **However, we do recommend running the latest version** of the Mattermost Server as it contains the most recent features and any applicable security updates.
+
+    If this is not possible, we encourage you to bon at least version 5.12 of the Mattermost Server. This release has a number of critical feature updates that will ensure compatibility in a number of areas, including the Mattermost push notification service (MPNS).
```

Comment:
```suggestion
    If this is not possible, it's recommended that you install at least version 5.12 of Mattermost Server. This release has a number of critical feature updates that will ensure compatibility in a number of areas, including the Mattermost push notification service (MPNS).
```

Reply:
Other than the typo what's the thought behind the change?

Reply:
It's based on the stopwords hackathon project and the avoidance of phrases such as "we recommend" in favor of "it's recommended" and "best practice is to". The opposite of "encourage" is "discourage" which is negative and best-avoided. 

4/5 on this and happy to discuss further. :)

Reply:
I disagree and have belabored this point: first person is not bad. This is not intended to be proper product documentation, but is living here for lack of a current home. We (Mattermost) are making a recommendation. Using the pronoun reduces this impact. Furthermore, this guide has taken on some close review that current documentation has not. Again, direct from the referenced documentation "... we recommend using an EMM provider that supports Managed App Configuration. Many app wrapping toolkits, for example Citrix MDX, also provide Managed App Configuration options. We recommend contacting your provider for more information on what deployment options are available." https://docs.mattermost.com/mobile/mobile-overview.html